model/user/db.go

```diff
@@ -336,9 +341,55 @@ func GetOrCreateUser(userId, displayName, email, accessToken, refreshToken strin
 		return nil, errors.Wrapf(err, "decoding user '%s'", userId)
 
 	}
+
+	if err := setSlackInformation(env, u); err != nil {
+		return nil, errors.Wrapf(err, "setting slack information for user '%s'", userId)
```

Comment:
Can we log + continue on error here? The Okta user manager requires only the DB modification above to succeed, so it seems like setting the Slack auth would be a nice-to-have.

Reply:
done